app-modules/assist-data-model/src/Filament/Resources/StudentResource/Pages/ListStudents.php

```diff
@@ -91,6 +105,8 @@ public function table(Table $table): Table
                     SubscribeBulkAction::make(),
                     BulkEngagementAction::make(context: 'students'),
                     DeleteBulkAction::make(),
+                    JoinCareTeamBulkAction::make(),
+                    LeaveCareTeamBulkAction::make(),
```

Comment:
What do you think about taking a similar approach to Subscriptions and introducing a toggle as opposed to independent join/leave actions?